salt/utils/msgpack_compat.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+'''
+Compatibility layer for ``msgpack`` serialization.
+
+The C implementation of objects cannot deal with ``dict`` or ``list`` like
+objects that actually are no subclasses of ``dict`` or ``list``. This
+compatibility layer takes care of converting such objects before passing them
+to ``msgpack``.
+
+..versionadded:: Nitrogen
+'''
+
+from __future__ import absolute_import
+import msgpack as _msgpack_orig
+from salt.ext import six as _six
```

Comment:
Not yet, but this is solvable by doing:
```
import salt.ext.six as six
```
This doesn't expose symbols. At least, we haven't any reports that it exposes anything except the module which is ignorable by loader.